tests/test_testproblems.py

```diff
@@ -91,3 +93,98 @@ def test_re24():
     for i in range(len(res)):
         obj_values = np.array([res[obj.symbol][i] for obj in problem.objectives])
         assert np.allclose(obj_values, expected_result[i])
+
+
+
+@pytest.mark.forest_problem
+def test_forest_problem():
```

Comment:
This test doesn't throw any errors on your side? I got error 
'TypeError: must be real number, not list' from desdeo/problem/gurobipy_evaluator.py", line 113

Reply:
No errors on my side, but by the looks of that error, my guess is that you don't have the updated versions of those gurobipy files. The changes were made in the closed pull request #121.